uber/tests/test_migration.py

```diff
@@ -0,0 +1,6 @@
+from uber.tests import dump_alembic_schema, dump_reset_uber_db_schema
+
+
+def test_alembic_migrations():
+    assert dump_alembic_schema() == dump_reset_uber_db_schema(), (
+        'Missing or invalid migrations, see alembic/README.md')
```

Comment:
Here's another neat trick. First we create the database using only our alembic migrations. Then we create the database in the usual way with `reset_uber_db()`. Finally we compare the schemas produced by both methods. If the schema's don't match, then we either forgot to create a migration or the migration we did create isn't doing what we expect.